src/store/getters/state-getters.ts

```diff
@@ -92,43 +96,53 @@ export const getEntityType = (state: StateIF): CorpTypeCd => {
 }
 
 /** Whether the entity is a Benefit Company. */
-export const isEntityTypeBEN = (state: StateIF): boolean => {
+export const isTypeBEN = (state: StateIF): boolean => {
```

Comment:
It would be good to make these common between Create/Filings/Edit UI.

Any thoughts on the preferred getter naming convention?
- Create UI -> isTypeBcCompany
- Filings UI -> isBcCompany
- Edit UI -> isTypeBC

Reply:
I like isBcCompany the best it is pretty clear what it is.